internal/pkg/services/argus/client/client.go

```diff
@@ -18,6 +18,7 @@ func ConfigureClient(p *print.Printer) (*argus.APIClient, error) {
 
 	authCfgOption, err := auth.AuthenticationConfig(p, auth.AuthorizeUser)
 	if err != nil {
+		p.Debug(print.ErrorLevel, "configure authentication: %v", err)
```

Comment:
This change should be done for all of the other clients

Reply:
Edited debug messages in all files